apps/fares/lib/proposed_locations/location.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Fares.ProposedLocations.Location do
+  @moduledoc """
+  Proposed Sales Location structure that matches the data coming from ArcGIS
```

Comment:
We might want to limit this just to fields we care about, for simplicity - we probably aren't going to use FFFVMUpdat, LFFVMUpdat, RSTUpdat, why_sn.